js/controller.js

```diff
@@ -1745,6 +1745,8 @@ module.exports = function(OO, _, $, W) {
       this.state.cast.showButton = this.isChromecastEnabled(params);
       this.state.cast.isReceiver = params.chromecast && params.chromecast.isReceiver;
 
+      this.state.isAirplayAllowed = Utils.getPropertyValue(params, 'airplay', true);
```

Comment:
Do we have to call the function or we have the value in this.state.playerParam.airplay?

Reply:
Yes we should have this value in the this.state.playerParam.airplay, and I used this function just for the default value.